src/test/java/org/testeditor/fixture/host/HostDriverFixtureTest.java

```diff
@@ -17,28 +19,27 @@
   private final int MAX_ROWS = 24;
   private final int MAX_COLUMNS = 80;
   private final String STANDARD_WINDOW_ID = "0x0";
+  private final String S2370_PATH = System.getenv("S3270_PATH");
+  private final String hostUrl = System.getenv("HOST_URL");
+  private final int hostPort = Integer.parseInt(System.getenv("HOST_PORT"));
+  HostDriverFixture hdf;
```

Comment:
why not name it 'hostDriverFixture', we don't use this variable too much, so the abbreviation does not help, does it?